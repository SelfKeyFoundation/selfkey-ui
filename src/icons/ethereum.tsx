import * as React from 'react';
import { WithStyles, withStyles, createStyles } from '@material-ui/core';
import { SvgIcon } from '@material-ui/core';
import { SvgIconProps } from '@material-ui/core/SvgIcon';

const styles = createStyles({});

export const EthereumIcon = withStyles(styles)((props: SvgIconProps & WithStyles<typeof styles>) => (
    <SvgIcon style={{ width: props.width || '44px', height: props.height || '44px' }} {...props} viewBox={props.viewBox || '0 0 44 44'}>
        <g fill="none" fillRule="evenodd">
        <rect width="44" height="44" fill="#9418DC" rx="9"/>
        <path fill="#FFF" fillRule="nonzero" d="M11.37 20.286l10.027-4.1 9.719 4.205L21.382 5.5 11.37 20.286zm.031 4.55l10.012 5.627 10.212-5.627L21.505 38.5 11.401 24.836zm10.043-6.645l10.15 4.385-10.15 5.508L11 22.456l10.444-4.265z"/>
        </g>
    </SvgIcon>
));

export default EthereumIcon;
