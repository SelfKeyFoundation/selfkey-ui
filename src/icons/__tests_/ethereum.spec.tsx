import * as React from 'react';
import { create } from 'react-test-renderer';
import { EthereumIcon } from '../ethereum';

describe('Ethereum Icon', () => {
	it('should show Ethereum Icon', () => {
		const component = create(<EthereumIcon />);
		const tree = component.toJSON();
		expect(tree).toMatchSnapshot();
	});
});
