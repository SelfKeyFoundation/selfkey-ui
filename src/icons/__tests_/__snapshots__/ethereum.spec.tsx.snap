// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Ethereum Icon should show Ethereum Icon 1`] = `
<svg
  aria-hidden="true"
  className="MuiSvgIcon-root Component-root-0-2-1"
  focusable="false"
  viewBox="0 0 44 44"
>
  <g
    fill="none"
    fillRule="evenodd"
  >
    <rect
      fill="#9418DC"
      height="44"
      rx="9"
      width="44"
    />
    <path
      d="M11.37 20.286l10.027-4.1 9.719 4.205L21.382 5.5 11.37 20.286zm.031 4.55l10.012 5.627 10.212-5.627L21.505 38.5 11.401 24.836zm10.043-6.645l10.15 4.385-10.15 5.508L11 22.456l10.444-4.265z"
      fill="#FFF"
      fillRule="nonzero"
    />
  </g>
</svg>
`;
